runtime-modules/content/src/lib.rs

```diff
@@ -710,22 +710,23 @@ decl_module! {
                 Storage::<T>::upload_data_objects(params)?;
             }
 
-            let nft_status = params.auto_issue_nft
-                .as_ref()
-                .map_or(
-                    Ok(None),
-                    |issuance_params| {
-                        Some(Self::construct_owned_nft(issuance_params)).transpose()
-                    }
-            )?;
+            // TODO AUDIT3: enable after open auction fix
+            // let nft_status = params.auto_issue_nft
+            //     .as_ref()
+            //     .map_or(
+            //         Ok(None),
+            //         |issuance_params| {
+            //             Some(Self::construct_owned_nft(issuance_params)).transpose()
+            //         }
+            // )?;
 
             // create the video struct
             let video: Video<T> = VideoRecord {
                 in_channel: channel_id,
                 is_censored: false,
                 enable_comments: params.enable_comments,
                 video_post_id:  None,
-                nft_status,
+                nft_status: None,
```

Comment:
Please, add the TODO comment here as well.

Reply:
fixed in 93de823c63787575d87f47916ccd33f7530469be